lib/builder/step/copy_step_test.go

```diff
@@ -44,7 +44,7 @@ func abs(path string) string {
 func TestNewCopyStep(t *testing.T) {
```

Comment:
I think there need to be a new unit test here.

Reply:
don't think so... I think they are closely coupled just like `chown`